Library/Formula/openhmd.rb

```diff
@@ -0,0 +1,32 @@
+class Openhmd < Formula
+  homepage "http://openhmd.net"
+  url "http://openhmd.net/releases/openhmd-0.1.0.tar.gz"
+  sha1 "186c747399bd9a509ac8300acbae8823fc4fcc79"
+
+  head do
+    url "https://github.com/OpenHMD/OpenHMD.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "hidapi" => :build
+
+  def install
+    args = ["--prefix", prefix,
+            "--disable-debug",
+            "--disable-silent-rules",
+            "--disable-dependency-tracking",
+            "--bindir=#{prefix}/bin"]
+
+    system "./autogen.sh" if build.head?
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system "#{prefix}/bin/unittests"
```

Comment:
I was considering the addition of the ln -s of 

Cellar/openhmd/HEAD/include/openhmd/openhmd.h
to 
Cellar/python3/3.4.1_1/Frameworks/Python.framework/Versions/3.4/include/python3.4m/

to build python-rift developed @ https://github.com/lubosz/python-rift

Do you think it's worth adding it to the formula as an option or is it too much specific?

David